### PR TITLE
Fix db documentation

### PR DESCRIPTION
**Pull request by SUPERCILEX**
@samtstern It looks like @puf forgot the second `*` making `onDataChanged` appear to have no documentation. Oops! :smile:

### CODE REVIEW

File: database/src/main/java/com/firebase/ui/database/FirebaseListAdapter.java [hunk 1]
```diff
@@ -85,18 +85,6 @@ public void onDataChanged() {
...
-    /* This method will be triggered each time updates from the database have been completely processed.
-     * So the first time this method is called, the initial data has been loaded - including the case
-     * when no data at all is available. Each next time the method is called, a complete update (potentially
-     * consisting of updates to multiple child items) has been completed.
-     * <p>
-     * You would typically override this method to hide a loading indicator (after the initial load) or
-     * to complete a batch update to a UI element.
-     */
-    protected void onDataChanged() {
-    }
```
Comment (SUPERCILEX) [inline database/src/main/java/com/firebase/ui/database/FirebaseListAdapter.java]: Methods should not be positioned before constructors.

Comment (puf) [inline database/src/main/java/com/firebase/ui/database/FirebaseListAdapter.java]: I thought I added it near the end, didn't I?

Comment (SUPERCILEX) [inline database/src/main/java/com/firebase/ui/database/FirebaseListAdapter.java]: Nope, you added it before a constructor at [line 98](https://github.com/firebase/FirebaseUI-Android/pull/512/files/87b1c312be04e73de30d556027ec1fbd35a8dfd7#diff-6cb76cf605058d8e425bb77017b102d2R98). I put between `onCancelled`.

**Comment by samtstern** [comment]
LGTM, good catch!

**Pull request merged**